### PR TITLE
Make changing port from cli possible.

### PR DESCRIPTION
**Pull request by UncleBill**
Can pass optional argument to `npm run dev`. like:

`npm run dev -- --port 6060` run at port 6060.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,12 @@ $ npm install -g vue-cli
 $ vue init webpack my-project
 $ cd my-project
 $ npm install
+# run at default port 8080
 $ npm run dev
+# run at port 6060
+$ npm run dev -- --port 6060
 ```
 
-If port 8080 is already in use on your machine you must change the port number in `/config/index.js`. Otherwise `npm run dev` will fail.
```
Comment (posva) [inline README.md]: As this is still true, can you leave something similar like:

```
You can change the port at `config/index.js` as well if you don't want to pass the option every time
```

**Comment by UncleBill** [comment]
@posva  updated.

File: README.md [hunk 2]
```diff
@@ -18,10 +18,13 @@ $ npm install -g vue-cli
...
+You can change the port at `config/index.js` as well.
```
Comment (posva) [inline README.md]: _if you don't want to pass the option every time_

**Comment by posva** [comment]
Actually, this is not necessary once #913 is merged

**Comment by UncleBill** [comment]
Well, port detection is more intelligent, but my patch is very light without extra dependency, it provides chance to specify a port anytime .

**Comment by LinusBorg** [comment]
Hi @UncleBill 

Thanks for your PR. However, I just merged port detection, also you can use the PORT env variable as well as the config. That should provide more than enough options to configure this.

So I have to close this PR.

**Pull request closed without merging**